File: class04/java/AP0402-Exemplo01.java
public class TabelaVerdadeAND {
    public static void main(String[] args) {
        System.out.println("A     | B     | A AND B");
        System.out.println("------+-------+-------");
        boolean[] vals = {true, false};
        for (boolean a : vals)
            for (boolean b : vals)
                System.out.printf("%-5s | %-5s | %s%n", a, b, a && b);
    }
}

File: class04/cpp/AP0402-Exemplo01.cpp
#include <iostream>
#include <iomanip>
int main() {
    bool vals[] = {true, false};
    std::cout << "A     | B     | A AND B" << std::endl;
    std::cout << "------+-------+-------" << std::endl;
    for (bool a : vals)
        for (bool b : vals)
            std::cout << std::boolalpha << std::left << std::setw(5) << a << " | " << std::setw(5) << b << " | " << (a && b) << std::endl;
    return 0;
}